files/stopsovetnik/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "StopSovetnik.ru",
+  "repo": "teplov/stopsovetnik",
+  "files": {
+    "include": ["lib/ss.min.js", "lib/ss.js"]
```

Comment:
There are no tags here https://github.com/teplov/stopsovetnik/releases
The bot will fail